sudoku_gen.py

```diff
@@ -94,9 +94,21 @@ def createPopulation(sudoku: Sudoku) -> List[Solution]:
     """
     P = []
     for _ in range(N_POPULATION):
-        grid = []
-        # TODO: tam gdzie są zera w planszy (czyli pola puste) przypisujemy przypadkowe wartości z zakresu 1-9, ale tak żeby w każdym bloku 3x3 nie było powtórzeń
-        P += (Solution(grid))
+        new_grid = np.copy(sudoku.grid)
+        for i in range(0, 9, 3):
+            for j in range(0, 9, 3):

```

Comment:
jest fukcja giveSubgrids - myślę że ładniej by to z tym wyglądało, ale jak nie to może zostać tak jak jest :)

Reply:
zrobione, zawsze jakiś zysk na czytelności